regolith/schemas.py

```diff
@@ -1307,9 +1396,11 @@
             "required": False,
             "type": "string",
         },
+        # TODO: Can this be required only if status = active?
```

Comment:
I think so.  CC @CJ-Wright

Reply:
Hmm I think this would work in reverse. You can do something similar to the code here: https://github.com/pyeve/cerberus/issues/274#issuecomment-262670804 
In this case do an `anyof` with `True` and `False` and set `True`'s dependencies so that all the needed fields must be supplied to be active.